packages/slate-react/src/components/editable.tsx

```diff
@@ -523,6 +532,17 @@ export const Editable = (props: EditableProps) => {
             break
           }
         }
+
+        // Restore the actual user section if nothing manually set it.
+        const toRestore = EDITOR_TO_USER_SELECTION.get(editor)?.unref()
```

Comment:
Instead of the `EDITOR_TO_USER_SELECTION` map, can you use a regular variable here as it's in the same scope? Or am I missing something

Reply:
It's used [here](https://github.com/ianstormtaylor/slate/blob/f564a1c16325e68662b740231b4997bdf0b052c7/packages/slate-react/src/plugin/with-react.ts#L78-L83) to unset the user selection in case of a 'set_selection' operation.